builder/vmware/common/driver.go

```diff
@@ -175,6 +186,12 @@ func NewDriver(dconfig *DriverConfig, config *SSHConfig, vmName string) (Driver,
 		drivers = []Driver{esxiDriver}
 
 	} else {
+		var playerInstalled bool
+		_, err := playerFindVmplayer()
```

Comment:
Only thing I wonder, if you have both Workstation and Player installed, this change makes `player` the default if found, if this voluntary? It might be irrelevant though, feel free to disregard if so!

Reply:
Thinking about this a little more, this does introduce an issue since Player is also installed alognside Workstation. So, I'm going to back out this change.

Reply:
These changes are backed out in fc089e5.

I'll run some tests next week and report back here before we proceed with any merge.